### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nsitkoff**
@nathansitkoff

Some changes to some stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # hello-world
 Just another repository
+
+things things stuff stuff
```
Comment (foobarzalot) [inline README.md]: I hate those things and that stuff!  Fix it, please.

**Review by foobarzalot — CHANGES_REQUESTED** [review]
Please try again.

**Comment by nsitkoff** [comment]
@nathansitkoff is this better?

**Review by foobarzalot — APPROVED** [review]
OK, cool.

**Pull request merged**